internal/hammer/diff.go

```diff
@@ -611,7 +611,7 @@ func (g *Generator) primaryKeyEqual(x, y *Table) bool {
 }
 
 func (g *Generator) columnDefEqual(x, y spansql.ColumnDef) bool {
-	return cmp.Equal(x, y, cmpopts.IgnoreTypes(spansql.Position{}))
+	return cmp.Equal(x, y, cmpopts.IgnoreTypes(spansql.Position{}), cmpopts.IgnoreUnexported(spansql.TimestampLiteral{}))
```

Comment:
If I ignore changes here, won't changes to the default value be ignored?

I thought the code would be much better if it could be changed to pass the following test cases.

```go
			from: `
CREATE TABLE Nonces (
  nonce INT64 NOT NULL,
  expires TIMESTAMP NOT NULL DEFAULT(TIMESTAMP '2000-01-01 00:00:00.000000+00:00'),
) PRIMARY KEY(nonce);
`,
			to: `
CREATE TABLE Nonces (
  nonce INT64 NOT NULL,
  expires TIMESTAMP NOT NULL DEFAULT(TIMESTAMP '2000-01-01 12:00:00.000000+00:00'),
) PRIMARY KEY(nonce);
`,
			ignoreAlterDatabase: true,
			expected: []string{
				`ALTER TABLE Nonces ALTER COLUMN expires TIMESTAMP NOT NULL DEFAULT (TIMESTAMP '2000-01-01 12:00:00.000000+00:00')`,
			},
```

This is an example of an approach, but you might want to make the following comparison here and modify it so that subsequent code will issue an alter statement for changes in the default value.

```go
cmp.Comparer(func(x, y spansql.TimestampLiteral) bool {
	return time.Time(x).Equal(time.Time(y))
}))
```

Reply:
My bad…

Ok, I've improved the code.
Could you check it again?

Reply:
Sorry, it seems I had rerun only a failed test at that time. 🙇 
Now the test runs in success locally.